WalletWasabi.Fluent/ViewModels/WalletManagerViewModel.cs

```diff
@@ -88,13 +91,13 @@ public WalletManagerViewModel(WalletManager walletManager, UiConfig uiConfig, Bi
 					wallet =>
 				{
 					WalletViewModelBase vm = (wallet.State <= WalletState.Starting)
-						? ClosedWalletViewModel.Create(walletManager, wallet)
+						? ClosedWalletViewModel.Create(this, wallet, legalChecker)
 						: WalletViewModel.Create(uiConfig, wallet);
 
 					InsertWallet(vm);
 				});
 
-			Dispatcher.UIThread.Post(() => LoadWallets(walletManager));
+			Dispatcher.UIThread.Post(() => LoadWallets(walletManager, legalChecker));
```

Comment:
BTW what is this?

Reply:
Renamed to EnumerateWallets, its scheduling enumeration of the wallets list to fill the navbar with wallets.

now uses RxApp.MainThreadScheduler.